### PR TITLE
🚨🚨[Whisper Tok] Update integration test

### PR DESCRIPTION
**Pull request by sanchit-gandhi**
# What does this PR do?

The merges for the Whisper tokenizers were updated on the Hub in [this PR](https://huggingface.co/openai/whisper-tiny/discussions/40). While this is a breaking change, it is a required fix to ensure we have parity with the original OpenAI repo.

This PR updates the integration tests for the Whisper tokenizer to reflect the merge changes.

### CODE REVIEW

File: tests/models/whisper/test_tokenization_whisper.py [hunk 1]
```diff
@@ -67,26 +67,26 @@ def test_full_tokenizer(self):
         tokenizer = WhisperTokenizer.from_pretrained(self.tmpdirname)
 
         tokens = tokenizer.tokenize("This is a test")
-        self.assertListEqual(tokens, ["This", "Ġis", "Ġa", "Ġ", "test"])
+        self.assertListEqual(tokens, ["This", "Ġis", "Ġa", "Ġtest"])
 
         self.assertListEqual(
             tokenizer.convert_tokens_to_ids(tokens),
-            [5723, 307, 257, 220, 31636],
+            [5723, 307, 257, 1500],
```
Comment (sanchit-gandhi) [inline tests/models/whisper/test_tokenization_whisper.py:74]: This now gives equivalent results to the original:
```python
from whisper.tokenizer import get_tokenizer

tokenizer = get_tokenizer(True)
tokens = tokenizer.encode("This is a test")
print(tokens)
```
**Print Output:**
```
[5723, 307, 257, 1500]
```

File: tests/models/whisper/test_tokenization_whisper.py [hunk 2]
```diff
@@ -499,25 +499,3 @@ def test_offset_decoding(self):
 
         output = multilingual_tokenizer.decode(INPUT_TOKENS, output_offsets=True)["offsets"]
         self.assertEqual(output, [])
-
-    @require_jinja
-    def test_tokenization_for_chat(self):
```
Comment (sanchit-gandhi) [inline tests/models/whisper/test_tokenization_whisper.py:504]: Chat template doesn't make sense for Whisper (a speech recognition model) - have removed the test to keep the CI lightweight (cc @Rocketknight1)

Comment (Rocketknight1) [reply tests/models/whisper/test_tokenization_whisper.py:504]: Fine with me!

**Review by ArthurZucker — APPROVED** [review]
Thanks for the prompt fix, it's breaking so I'll probably update the PR tittle with ⚠️

**Comment by sanchit-gandhi** [comment]
The GH PR itself is not strictly breaking (there's no change to the code), but rather it's the Hub PR which is breaking. Fine for me to leave the 🚨 in the title though to book-log this!

**Pull request merged**